lib/corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
+  def correct_name(text)
+    text.capitalize[0..9]
```

Comment:
Please use `#capitalize!` instead of `#capitalize` because you need to change the original variable. `#capitalize` does not change the variable.
The method should trim the word if the name has more than 10 characters